From abcb449a6f7f5c70e9909eb87718e4ca6fbc5c79 Mon Sep 17 00:00:00 2001
From: YunYi Yang <yangyunyi2@huawei.com>
Date: Mon, 20 Nov 2023 10:16:31 +0800
Subject: [PATCH 107/108] config: arm64: Enable dubugfs config of hisi sas

driver inclusion
category: bugfix
bugzilla: https://gitee.com/src-openeuler/kernel/issues/I8F81L

----------------------------------------------------------------------------

Enable dubugfs config of hisi sas. Set
CONFIG_SCSI_HISI_SAS_DEBUGFS_DEFAULT_ENABLE to y.

Signed-off-by: YunYi Yang <yangyunyi2@huawei.com>
---
 arch/arm64/configs/openeuler_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index b89c386c1bc7..f3b055bc18f0 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -2134,6 +2134,7 @@ CONFIG_SCSI_AACRAID=m
 # CONFIG_SCSI_AIC94XX is not set
 CONFIG_SCSI_HISI_SAS=m
 CONFIG_SCSI_HISI_SAS_PCI=m
+CONFIG_SCSI_HISI_SAS_DEBUGFS_DEFAULT_ENABLE=y
 # CONFIG_SCSI_MVSAS is not set
 # CONFIG_SCSI_MVUMI is not set
 # CONFIG_SCSI_ADVANSYS is not set
-- 
2.27.0

